### DISCUSSION TITLE
Unofficial yarr Docker Image

### DISCUSSION THREAD

**Discussion by versun** [nkanaev/yarr#202] [Announcements]
Hi yarr Team,

Since yarr doesn't provide an official Docker image, I created a Dockerfile and docker-compose.yml to build an unofficial one. 
It's available in my repo: [https://github.com/versun/yarr](https://github.com/versun/yarr)

Feel free to use it as a reference for creating an official yarr Docker image. An official image would be great for Docker users.

Let me know if you have any questions!

Thanks,
Versun

**Comment by tillcash** [comment]
Shout out to https://github.com/Wakeful-Cloud/yarr

**Comment by nkanaev** [comment]
`docker pull ghcr.io/nkanaev/yarr`

**Comment by nkanaev** [comment]
Official docker images are available now on both Docker Hub and GitHub Container Registry now, see https://nkanaev.github.io/yarr/en/installation/docker/ for more info.
